x-pack/plugins/searchprofiler/public/components/searchprofiler_tabs.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
```

Comment:
Couple extra newlines here — strange the linter didn’t catch it; I think it is configured to only allow 2 consecutive empty lines.